packages/styles/src/components/form-footer.scss

```diff
@@ -0,0 +1,36 @@
+@forward './../variables/options';
+@use './../themes/bootstrap/core' as *;
+
+@use '../variables/color';
+@use '../variables/commons';
+@use '../variables/spacing';
+@use './../variables/breakpoints';
+@use '../mixins/utilities' as utility-mx;
+@use '../tokens/temp-form-footer' as components;
+@use '../functions/tokens' as tokens;
+
+tokens.$default-map: components.$post-form-footer;
+
+.form-footer {
+  border-top: tokens.get('post-form-footer', 'post-form-footer-border-top-width') solid
+    tokens.get('post-form-footer', 'post-form-footer-border-top-color');
+  padding-top: tokens.get('post-form-footer', 'post-form-footer-padding-top-s');
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: tokens.get('post-form-footer', 'post-form-footer-gap');
+
+  &-right-actions {
+    @include utility-mx.responsive-actions();
+    gap: tokens.get('post-form-footer', 'post-form-footer-gap');
+
+    @include media-breakpoint-up(md) {
+      margin-left: auto;
+    }
+  }
+
+  @include media-breakpoint-up(md) {
+    padding-top: tokens.get('post-form-footer', 'post-form-footer-padding-top-m');
```

Comment:
The tokens are CSS custom properties that are automatically updated for all breakpoints, you do not have to handle the responsiveness manually in the component styles.